docs/model_card.rst

```diff
@@ -58,15 +58,17 @@ template which includes the following slots for free text sections:
   out, in case of questions about the model or the model card.
 - ``"citation_bibtex"``: Bibtex style citations for the model or resources used
   to train the model.
+- ``"eval_methods"``: Details about evaluation process of the model.
 
 The template also contains the following sections that are automatically
 generated by ``skops``.
 
 - ``"hyperparameter_table"``: Hyperparameters of the model.
 - ``"model_plot"``: A diagram of the model, most relevant in case the model is
   a complex scikit-learn :class:`~sklearn.pipeline.Pipeline`.
+- ``"eval_results"``: Table of evaluation results.
```

Comment:
I'd move this beside `eval_methods`. This is not automatically generated w/o user doing something anyway. It can be moved up, and mention that `add_metrics` creates this thing.